docs/management_api.yml

```diff
@@ -387,9 +387,11 @@ paths:
           $ref: "#/responses/InternalServerError"
 
     post:
-      summary: Create an artifact
+      summary: Upload mender artifact
       description: |
-        Creates artifact. Mulitpart request with meta and artifact.
+        Upload medner artifact. Mulitpart request with meta and artifact.
```

Comment:
I suppose this is in master as well, but nevertheless:
`s/medner/mender/`
`s/Mulitpart/Multipart/`